packages/serverless/src/domain/use-case/document-history.ts

```diff
@@ -89,11 +111,11 @@ async function listHistoryDocuments(
 
     const pipeline = buildHistoryPipeline(pagination);
 
-    const documentsWithMetadata = await documentsCollection
+    const documentsWithMetadata = (await documentsCollection
       .aggregate(pipeline)
-      .toArray();
+      .toArray()) as TDocumentHistorySerialized[];
```

Comment:
I'm not prefer type cast, some case it would create more problem than it solved.

Reply:
this is returned by database query, I already noted a TODO to debug this type annotation later when serverless can be built successfully